jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildService.java

```diff
@@ -260,6 +225,18 @@ protected String updateOrCreateBuildConfig(BuildServiceConfig config, OpenShiftC
         }
     }
 
+    private void initClient() {
+        ClusterAccess clusterAccess = jKubeServiceHub.getClusterAccess();
+        if (clusterAccess == null) {
+            clusterAccess = new ClusterAccess(log,
+                ClusterConfiguration.from(System.getProperties(), jKubeServiceHub.getConfiguration().getProject().getProperties()).build());
+        }
+        client = clusterAccess.createDefaultClient();
```

Comment:
assigning value of `clusterAccess.createDefaultClient()` which can either be `DefaultKubernetesClient` or `DefaultOpenShiftClient` seems to `OpenShiftClient client` seems to be causing https://github.com/eclipse/jkube/issues/815 

In v1.3.0, this code was in JKubeServiceHub and `client` was of type `KubernetesClient`